packages/manager/modules/iam/src/components/highlightText/highlightText.styles.scss

```diff
@@ -0,0 +1,3 @@
+.iam-highlight-text {
+  background: rgb(255, 230, 0);
```

Comment:
**nitpick:** Use the color available from ui-kit utilities.
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-backgrounds--primary